DocumentDBStudio/MainForm.Designer.cs

```diff
@@ -28,1020 +28,1022 @@ protected override void Dispose(bool disposing)
-            this.btnUpdateEntity = new System.Windows.Forms.ToolStripButton();
-            this.tsbMergeEntity = new System.Windows.Forms.ToolStripButton();
-            this.btnDeleteEntity = new System.Windows.Forms.ToolStripButton();
-            this.toolStripSeparator3 = new System.Windows.Forms.ToolStripSeparator();
-            this.tsbEditTemplate = new System.Windows.Forms.ToolStripButton();
-            this.settingsToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
-            this.toolStripSeparator4 = new System.Windows.Forms.ToolStripSeparator();
-            this.statusStrip.SuspendLayout();
-            ((System.ComponentModel.ISupportInitialize)(this.splitContainerOuter)).BeginInit();
-            this.splitContainerOuter.Panel1.SuspendLayout();
-            this.splitContainerOuter.Panel2.SuspendLayout();
-            this.splitContainerOuter.SuspendLayout();
-            ((System.ComponentModel.ISupportInitialize)(this.splitContainerInner)).BeginInit();
-            this.splitContainerInner.Panel1.SuspendLayout();
-            this.splitContainerInner.Panel2.SuspendLayout();
-            this.splitContainerInner.SuspendLayout();
-            this.tabControl.SuspendLayout();
-            this.tabRequest.SuspendLayout();
-            this.tabCrudContext.SuspendLayout();
-            ((System.ComponentModel.ISupportInitialize)(this.splitContainerIntabPage)).BeginInit();
-            this.splitContainerIntabPage.Panel1.SuspendLayout();
-            this.splitContainerIntabPage.Panel2.SuspendLayout();
-            this.splitContainerIntabPage.SuspendLayout();
-            this.tabResponse.SuspendLayout();
-            this.tabPageRequestOptions.SuspendLayout();
-            this.groupBox3.SuspendLayout();
-            this.groupBox2.SuspendLayout();
-            this.groupBox1.SuspendLayout();
-            this.tabDocumentCollection.SuspendLayout();
-            this.groupBox4.SuspendLayout();
-            this.tabOffer.SuspendLayout();
-            this.gbStandardOffer.SuspendLayout();
-            ((System.ComponentModel.ISupportInitialize)(this.ButtomSplitContainer)).BeginInit();
-            this.ButtomSplitContainer.Panel1.SuspendLayout();
-            this.ButtomSplitContainer.Panel2.SuspendLayout();
-            this.ButtomSplitContainer.SuspendLayout();
-            this.triggerPanel.SuspendLayout();
-            this.feedToolStrip.SuspendLayout();
-            this.tsMenu.SuspendLayout();
-            this.tsAddress.SuspendLayout();
-            this.menuStrip1.SuspendLayout();
-            this.tsTablesEntities.SuspendLayout();
-            this.SuspendLayout();
-            // 
-            // statusStrip
-            // 
-            this.statusStrip.Items.AddRange(new System.Windows.Forms.ToolStripItem[] {
+			System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(MainForm));
```

Comment:
Sam as before. I'm guessing it's auto generated, but it'd undesirable to have this indenting and the change.